scripts/subquery-e2e.py

```diff
@@ -121,3 +121,23 @@
 """
 Legacy Bridge Swap
 """
+
+contract = LedgerContract("../../eth-bridge/contracts/fetch/bridge.wasm", ledger_client)
```

Comment:
I guess this is not really going to work for other users. I think that I would probably reference the released version here:

https://github.com/fetchai/fetch-ethereum-bridge-v1/releases/tag/v0.2.0

The script could create a hidden `.contracts` folder in the project root and download it if the contract was not present or wrong checksum etc.

What are your thoughts?

Reply:
Ah yep, my mistake - hadn't noticed that one!

Reply:
Just pushed a quick fix, let me know if this one is too simplistic - apologies, have been a bit wrapped up in the NativeTransfer stuff since this review.